docs/deployment/configuration/resource_manager.rst

```diff
@@ -0,0 +1,113 @@
+.. _deployment-configuration-resource-manager:
+
+#####################
+Flyte ResourceManager
+#####################
+
+**Flyte ResourceManager** is a configurable component that allows plugins to manage resource allocations independently. It helps track resource utilization of tasks that run on Flyte. The default deployments are configured as ``noop``, which indicates that the ResourceManager provided by Flyte is disabled and plugins rely on each independent platform to manage resource utilization. In situations like the K8s plugin, where the platform has a robust mechanism to manage resource scheduling, this may work well. However, in a scenario like a simple web API plugin, the rate at which Flyte sends requests may overwhelm a service and benefit from additional resource management.
+
+The below attribute is configurable within FlytePropeller, which can be disabled with:
+
+.. code-block:: yaml
+
+    resourcemanager:
+      type: noop
+
+The ResourceManager provides a task-type-specific pooling system for Flyte tasks. Optionally, plugin writers can request resource allocation in their tasks.
+
+A plugin defines a collection of resource pools using its configuration. Flyte uses tokens as a placeholder to represent a unit of resource.
+
+How does a Flyte plugin request for resources?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Flyte plugin registers the resource and the desired quota of every resource with the **ResourceRegistrar** when setting up FlytePropeller. When a plugin is invoked, FlytePropeller provides a proxy for the plugin. This proxy facilitates the plugin's view of the resource pool by controlling operations to allocate and deallocate resources.
+
+.. dropdown:: :fa:`info-circle` Enabling Redis instance
```

Comment:
I would recommend moving this to the bottom of the section, below all the other content, and either changing the dropdown title to "Enabling a Redis instance as an external store to track and manage resource pool allocation" or turning the dropdown into a note, so people don't miss this info.